config/objects/rewardable.json

```diff
@@ -193,13 +214,38 @@
 			}
 		},
 		"types" : {
-			"object" : {
+			"flotsam" : {
+				//GENIE.WAV
 				"index" : 0,
 				"aiValue" : 2000,
 				"rmg" : {
 					"value"		: 2000,
 					"rarity"	: 100
-				}
+				},
+				"visitMode" : "unlimited",
+				"selectMode" : "player",
+				"rewards" : [
```

Comment:
One option is missing: https://heroes.thelazy.net/index.php/Flotsam

Reply:
Ah, I see it, 25% for 5 wood. Will fix.
Current json is based on ancient version of our code, will recheck.